src/lib/customers/20minuten-en.json

```diff
@@ -0,0 +1,54 @@
+	"background": "20 Minuten is Switzerland's largest commuter newspaper and one of the most visited websites in Switzerland. It is a media portal with daily news, entertainment and inspirational articles. In 2019, 20 Minuten had planned a major migration of its applications away from the legacy infrastructure and application code, some of which was almost 20 years old. This involved migrating existing VMs to a modern Kubernetes-based container infrastructure on AWS. The backend was to be migrated from Perl to Go and containerised. The frontend was also rendered by Perl and was to be converted into a modern Next.js application. 20 Minuten has over one million users every day. This makes a migration without downtime particularly challenging. In addition, it was important to keep the operating costs within manageable limits despite this large load.",
+	"goal": "The goal of the project was to modernize the setup of the application. The legacy components and the previous code were to be replaced in order to make the system more stable and cost-efficient. AWS was chosen as the infrastructure provider and the application was to be redeployed in Go, Node.js and React microservices on an EKS (Kubernetes) cluster.",
+	"contribution": {
+		"intro": "bespinian had a key role in the following topics regarding application migration in terms of refactoring / re-architecting from the 6 migration strategies of AWS.",
+		"topics": [
+			{
+				"title": "Infrastructure on AWS",
+				"body": "bespinian has been instrumental in provisioning the infrastructure on AWS. We played a key role in writing and applying the appropriate Terraform modules and setting up the Kubernetes clusters on which the various microservices are hosted. An important part of this was also the correct configuration of the caching layer (Amazon CloudFront) to keep the costs manageable with the large user:inside number of 20 Minuten. bespinian did a lot of the technical implementation and automation, but also the communication between the development and infrastructure teams."
+			},
+			{
+				"title": "Kubernetes and container configuration",
+				"body": "An exciting task was the configuration of the Kubernetes cluster (on Amazon EKS). The architecture as well as the implementation of it still bears bespinian's signature. It was about architectural decisions, but also about the implementation of the provisioning and configuration of the cluster via Terraform. Subsequently, it was about the automation of the deployments of the individual services via Helm and GitHub Actions. Bespinian was allowed to participate in all these sub-areas. We also played a major role in coordinating the individual teams and applications in the sense of a DevOps migration."
```

Comment:
I would replace `Bespinian was allowed to participate in all these sub-areas.` with `bespinian played a key role in all of these areas.`